core/chains/evm/txmgr/nonce_syncer.go

```diff
@@ -80,11 +79,8 @@ func NewNonceSyncer(orm ORM, lggr logger.Logger, ethClient evmclient.Client, kst
 //
 // This should only be called once, before the EthBroadcaster has started.
 // Calling it later is not safe and could lead to races.
-func (s NonceSyncer) Sync(ctx context.Context, keyState ethkey.State) (err error) {
-	if keyState.Disabled {
-		return errors.Errorf("cannot sync disabled key state: %s", keyState.Address)
-	}
-	err = s.fastForwardNonceIfNecessary(ctx, keyState.Address.Address())
+func (s NonceSyncer) Sync(ctx context.Context, address common.Address) (err error) {
+	err = s.fastForwardNonceIfNecessary(ctx, address)
```

Comment:
Why has the disabled check disappeared?

Reply:
I think it's because all addresses passed into this function are the result of the new revamped GetEnabledAddresses() which filters those out already

Reply:
Yes. The KeyStore interface no longer returns disabled addresses.